modules/post/windows/manage/install_python.rb

```diff
@@ -0,0 +1,84 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::Common
+  include Msf::Post::File
+  include Msf::Post::Windows::Powershell
+
+  def initialize(info = {})
+    super(update_info(info,
+                      'Name'          => 'Install Python for Windows',
+                      'Description'   => '
+                        This module places an embeddable Python3 interpreter onto the target file system,
+                        granting pentesters access to a lightweight Python interpreter.
+                        This module does not require administrative privileges or user interaction with
+                        installation prompts.
+                      ',
+                      'License'       => MSF_LICENSE,
+                      'Author'        => ['Michael Long <bluesentinel[at]protonmail.com>'],
+                      'Arch' => [ARCH_X86, ARCH_X64],
+                      'Platform'      => [ 'win' ],
+                      'SessionTypes'  => [ 'meterpreter', 'shell' ],
+                      'References'	=> [
+                        ['URL', 'https://docs.python.org/3/using/windows.html#windows-embeddable'],
+                        ['URL', 'https://attack.mitre.org/techniques/T1064/']
+                      ]))
```

Comment:
This should all be indented 6 spaces (2 more than line 12)

Reply:
Fixed